service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -87,33 +89,41 @@ public Users getUserPojo(long userId) {
   public boolean isValidLogin(String email, String pass) {
     Optional<Users> maybeUser =
         Optional.ofNullable(
-            db.selectFrom(USERS).where(USERS.EMAIL.eq(email)).fetchOneInto(Users.class));
+            db.selectFrom(USERS)
+                .where(USERS.EMAIL.eq(email).and(USERS.DELETED_AT.isNull()))
+                .fetchOneInto(Users.class));
 
     return maybeUser
         .filter(user -> Passwords.isExpectedPassword(pass, user.getPasswordHash()))
         .isPresent();
   }
 
   /**
-   * TODO: Refactor this method to take in a DTO / POJO instance Creates a new row in the USER table
    * with the given values.
    *
    * @throws EmailAlreadyInUseException if the given username and email are already used in the USER
    *     table.
    */
-  public UsersRecord createNewUser(
-      String email, String password, String firstName, String lastName) {
+  public UsersRecord createNewUser(NewUserRequest newUserRequest) {
+    String email = newUserRequest.getEmail();
+
     boolean emailUsed = db.fetchExists(db.selectFrom(USERS).where(USERS.EMAIL.eq(email)));
     if (emailUsed) {
       throw new EmailAlreadyInUseException(email);
     }
 
+    String password = newUserRequest.getPassword();
+    String firstName = newUserRequest.getFirstName();
+    String lastName = newUserRequest.getLastName();
+    Country country = newUserRequest.getCountry();
+
     UsersRecord newUser = db.newRecord(USERS);
     newUser.setEmail(email);
     newUser.setPasswordHash(Passwords.createHash(password));
     newUser.setFirstName(firstName);
     newUser.setLastName(lastName);
     newUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
```

Comment:
Are all the users going to be started off with the standard privilege level?

Reply:
Yes, I think all users should start with `STANDARD` and only and `ADMIN` can *upgrade* standard users to admins